src/Stratis.SmartContracts.CLR.Tests/AuctionTests.cs

```diff
@@ -2,6 +2,7 @@
 using System.Collections.Generic;
 using Stratis.SmartContracts.CLR.Serialization;
 using Stratis.SmartContracts.Networks;
+using Stratis.SmartContracts.RuntimeObserver;
```

Comment:
How come this was added without an apparent use for it in the file? Was the project not building or something?

Reply:
`Stratis.SmartContracts.RuntimeObserver.IGasMeter` and `Gas` replaced `Stratis.SmartContracts.Gas`

Reply:
Identical code, just got moved